doc/eclipse.html

```diff
@@ -335,6 +343,52 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
 </ul>
 </p>
 
+<h3><a name="eclipse-m2e-integration">10 - M2E Integration</a></h3>
+
+TestNG M2E Integration plugin was introduced since 6.9.10 as an optional plugin.
+It allows you to run your tests with System Properties or JVM settings, which are defined by maven-surefire-plugin or maven-failsafe-plugin of pom.xml, to be appended to the runtime TestNG process.
+
+Once this plugin installed, you can see a dedicate preference page (workspace level settings):
+<p align="center">
+<img border="0" src="http://testng.org/pictures/m2e-pref.png" />
+</p>
+or, project properties page (as name implied, can have project specific settings to override the workspace ones:
```

Comment:
or on the project properties page. You can override workspace settings with project specific ones.